src/ext/configuration.h

```diff
@@ -30,8 +31,22 @@ bool ddtrace_config_env_bool(ddtrace_string env_name, bool default_value TSRMLS_
 bool ddtrace_config_distributed_tracing_enabled(TSRMLS_D);
 bool ddtrace_config_trace_enabled(TSRMLS_D);
 
+#define DDTRACE_LONGEST_INTEGRATION_ENV_PREFIX_LEN 10  // "DD_TRACE_" FTW!
+#define DDTRACE_LONGEST_INTEGRATION_ENV_SUFFIX_LEN 23  // "_ANALYTICS_SAMPLE_RATE" FTW!
+#define DDTRACE_LONGEST_INTEGRATION_ENV_LEN                                              \
+    (DDTRACE_LONGEST_INTEGRATION_ENV_PREFIX_LEN + DDTRACE_LONGEST_INTEGRATION_NAME_LEN + \
+     DDTRACE_LONGEST_INTEGRATION_ENV_SUFFIX_LEN)
+
 // note: only call this if ddtrace_config_trace_enabled() returns true
 bool ddtrace_config_integration_enabled(ddtrace_string integration TSRMLS_DC);
+bool ddtrace_config_integration_enabled_ex(ddtrace_integration_name integration_name TSRMLS_DC);
+bool ddtrace_config_integration_analytics_enabled(ddtrace_string integration TSRMLS_DC);
+bool ddtrace_config_integration_analytics_enabled_ex(ddtrace_integration_name integration_name TSRMLS_DC);
+double ddtrace_config_integration_analytics_sample_rate(ddtrace_string integration TSRMLS_DC);
+double ddtrace_config_integration_analytics_sample_rate_ex(ddtrace_integration_name integration_name TSRMLS_DC);
```

Comment:
Does this one need exposed? I only saw it used to implement `ddtrace_config_integration_analytics_sample_rate`.

Reply:
Nice catch! I've also removed `ddtrace_config_integration_analytics_enabled_ex` since it doesn't need to be public either.